bin/travis-deploy.sh

```diff
@@ -54,7 +54,7 @@ docker build --build-arg NODE_ENV=development \
 # Login to Docker Hub
 echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
 
-# Push an image and all the tags
+# Push an image and all the events
```

Comment:
Why did we rename tags into events?

Reply:
I'm guessing it happened when replacing the abci term `tags` with `events` in this prior commit
https://github.com/dashevo/js-drive/pull/428/commits/404ac158f6e0342252fbccd28f5aeb88ba31808c